task/rhacs-m2m-authenticate/0.1/README.md

```diff
@@ -0,0 +1,66 @@
+# Red Hat Advanced Cluster Security Deployment Check Task
+
+Exchanges a service account token against a short-lived authorization token to configure a pipeline run using `roxctl`.
+
+## Prerequisites
+
+This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet) or [StackRox](https://www.stackrox.io).  It also requires configuration of a machine-to-machine integration.
+
+<https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
```

Comment:
This doesn't seem like the right link?

Reply:
This link is present across the README of all other RHACS tasks. I wanted to keep consistency here.
I suppose you'd like to see a link to the documentation of RHACS short-lived token configuration.